### ISSUE TITLE
Thumbnail doesn't refresh when watching new video in same tab

### ISSUE THREAD

**Issue by DibraSol** [gifnksm/youtube_thumbnail_viewer#7]
First of all, thanks for the great add-on. It's very clean and minimalist and before updating to 0.3.0 I had no problems.
Now when I keep watching videos in the same tab and check the thumbnail, it only shows the thumbnail of the very first video I clicked on. It seems like it doesn't grab the current URL's thumbnail or it doesn't refresh. Refreshing the page or opening a new tab fixes the problem.

This might not even be an issue specifically of this add-on but due to the many other add-ons I use that change YouTube's behavior (like Unhook YouTube), but I just wanted to ask and see if others have the same problem.

**Comment by GammelSami** [comment]
Yeah, before version 0.3.0 the extension "brute forced" the best thumbnail resolution by URL. Since v0.3.0 the extension uses the [OGP metadata](https://github.com/gifnksm/youtube_thumbnail_viewer/issues/5#issuecomment-1025074376) from the website's HTML but YouTube does not refresh the metadata when you click on an other video.

In theory the new way is the more correct one. But often times it doesn't work.

I think the easiest fix would be a rollback to version 0.2.0 or using YouTube's API. This would also fix #6.
But the dev is inactive since Feb and I don't have the time/knowledge.
